ui/src/app/utils/workspace-utils.tsx

```diff
@@ -21,13 +17,15 @@ export const isValidBilling = (workspace: Workspace): boolean => {
   const enableInitialCreditsExpiration =
     serverConfigStore.get().config.enableInitialCreditsExpiration;
   const isExpired =
+    enableInitialCreditsExpiration &&
```

Comment:
subtle logic change, but I think it makes this a little clearer

Reply:
I like it. Thanks for cleaning it up.